pkg/nacler/keyers/secureenclave/app_resources/Info.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Maybe add a super short README to this `app_resources` directory to document that these files are for testing purposes and not used in a real build? (Makes the bundle name/bundle executable values here more understandable)

Reply:
done